guides/contributor/membership.md

```diff
@@ -115,6 +115,12 @@ Triagers may be code contributors, but writing code is not a requirement for
 becoming a triager. Triagers are encouraged to be active participants in project
 meetings, chat rooms, and other discussion forums.
 
+Triagers belong to a specific [SIG](processes.md/#special-interest-groups-sigs).
+There can be multiple triager groups in a single SIG. For example, there can be
+different triager groups for different repositories owned by a SIG, or different
+triager groups for different areas of expertise (e.g. different languages in the
+documentation).
```

Comment:
```suggestion
documentation). A SIG can set up additional triager groups for organizational purposes to help make it easier to refer to a group of contributors, such as for ownership within a repository, through requesting PR reviews, or for contacting elsewhere on GitHub.
```
I don't think we absolutely need this since we're essentially restating the purpose of having a group to begin with, but a takeaway I've had from our discussions within the Collector SIG is that it may not be immediately obvious when to create a group vs. just referring directly to certain contributors in things like a CODEOWNERS file or for requesting PR reviews. I think explaining _why_ a SIG would create additional groups may help make that clearer.

Reply:
Added, thanks!